.gitignore

```diff
@@ -1,4 +1,5 @@
 .nyc_output
+.nyc_cache
```

Comment:
Or better yet, maybe we should put it in:

`./node_modules/.cache/nyc`, and encourage other modules to use `./node_modules/.cache/<module-name>` as well. Then `trash node_modules/.cache` becomes the de facto way to clear caches when troubleshooting.